.github/workflows/ci-frontend.yml

```diff
@@ -3,7 +3,7 @@ name: frontend
 on:
   push:
     branches:
-    tags:
+      - master
```

Comment:
I'm not sure about this one either. If master pushed first and after this, we make a release tag I guess it will be fine. However, what will happen if we commit master locally, tag it locally and push the tag?

Reply:
I don't think we need to run this workflow in any case you mentioned, because we already ran them on PR merges. That means it's unnecessary runs.